### DISCUSSION TITLE
No particles when I make a call to breakBlock

### DISCUSSION THREAD

**Discussion by JackM33** [Minestom/Minestom#993] [Help]
Hi whenever I make a call to breakBlock(player, position) it successfully breaks the block but no particles are displayed. Do I need to generate those particles myself as another packet? If so, what particle type should I specify or how might I go about doing this? I cannot find the broken particles of each block type in the particle class.

Thank you!

**Comment by MrGazdag** [comment]
The packet that you need to send is an [EffectPacket](https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/network/packet/server/play/EffectPacket.java). Look at the [protocol documentation](https://wiki.vg/Protocol#Effect) for this. (Effect ID 2001: Block Break + block break sounds)

There is also a `block` particle, whose extra data is a block state. It is a particle, that is generated from a block state. The effect uses that particle, but it has a different spawning algorithm than spawning particles manually; as it follows the block shape (i think).

**Comment by emortaldev** [comment]
The packet does seem to get sent, just not sure why players can't see it
Should be fixed on #996
